### PR TITLE
Extract Prompt Hint to pure function

### PR DESCRIPTION
**Pull request by kyle-rader-msft**
This logic was already static but doesn't need to live on the CommandAad so before we re-use it in other command moving to it's own static class to hold the static function.

### CODE REVIEW

**Review by goagain — DISMISSED** [review]
Actually, we don't have chance to see the prompt hint since we moved Web mode to browser.

**Comment by kyle-rader-msft** [comment]
> Actually, we don't have chance to see the prompt hint since we moved Web mode to browser.

This is true at the moment, but we want to continue using it and accepting it bceause
1. It's good for telemetry distinction and,
2. Improving the UX of prompts when they happen is still an important feature desired by developers so it will be on the AzureAuth roadmap :)

**Review by shalinikhare27 — APPROVED** [review]

File: src/AzureAuth/PromptHint.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /// <summary>
+    /// Functions for ensuring a prompt hint is displayed with an AzureAuth prefix.
+    /// </summary>
+    public static class PromptHint
+    {
+        /// <summary>
+        /// Azure Auth Display name for prompt hints.
+        /// </summary>
+        public const string AzureAuth = "Azure Auth";
```
Comment (reillysiemens) [inline src/AzureAuth/PromptHint.cs:14]: Should this actually have been `AzureAuth` this whole time? 🤔

**Review by reillysiemens — APPROVED** [review]

**Review by goagain — APPROVED** [review]

**Review by shalinikhare27 — APPROVED** [review]

**Review by reillysiemens — APPROVED** [review]

**Review by mvanchaa — APPROVED** [review]

**Comment by kyle-rader-msft** [comment]
Poor GitHub Actions 
image

**Pull request merged**